File: CarMaintanance/Model/PerfilesModulos.cs
using System.ComponentModel.DataAnnotations;

namespace CarMaintanance.Model
{
    public class PerfilesModulos : baseClass
    {
        [Key]
        public int ID { get; set; }
        public int ModuloID { get; set; }
        public virtual Modulos Modulos { get; set; }
        public int PerfilID { get; set; }
        public virtual Perfiles Perfil { get; set; }
    }
}

File: CarsMaitananceFront/MaintananceCar/src/app/Interfaces/PerfilesModulos.ts
import { Perfil } from "./Perfil";
import { ModulosDetalle } from './ModulosDetalle';

export interface PerfilesModulos {
    iD: number;
    moduloID: number;
    modulos: ModulosDetalle;
    perfil: Perfil;
    perfilID: number;
}
